### ISSUE TITLE
L.HTMLUtils.buildTable breaks in IE9 & IE8

### ISSUE THREAD

**Issue by jack-r-abbit** [humangeo/leaflet-dvf#56]
IE 8 & 9 seem to take issue with these lines in L.HTMLUtils.buildTable in leaflet-dvf.js:

1342:	thead.innerHTML = '<tr><th>Name</th><th>Value</th></tr>';

1363:	tbody.innerHTML += '<tr><td>' + property + '</td><td>' + value + '</td></tr>';

This error is thrown: Invalid target element for this operation.

I was able to fix the issue by explicitly creating the tr, th/td using the same method you created the rest of the table elements and then setting the innerHTML of the td/th. But there may also be a better way to do it. It certainly isn't as elegant but it works.

```javascript
		var thead_tr = L.DomUtil.create('tr', '', thead);
		var thead_values = ['Name','Value'];
		for (var i = 0, l = thead_values.length; i < l; i++) {
			var thead_th = L.DomUtil.create('th', '', thead_tr);
			thead_th.innerHTML = thead_values[i];
		}
```
and
```javascript
				var tbody_tr = L.DomUtil.create('tr', '', tbody);
				tbody_values = [property, value];
				for (var i = 0, l = tbody_values.length; i < l; i++) {
					var tbody_td = L.DomUtil.create('td', '', tbody_tr);
					tbody_td.innerHTML = tbody_values[i];
				}

```

**Comment by sfairgrieve** [comment]
Good catch!  I'll fix that based on your suggestions, or feel free to submit a pull request.  Appreciate the fix!

**Comment by sfairgrieve** [comment]
I've incorporated these changes in the classification branch and will merge them in soon.
